src/components/Navbar.js

```diff
@@ -84,8 +84,8 @@ const Navbar = class extends React.Component {
             <Link className="navbar-item" to="/create-project">
               Create a Project
             </Link>
-            <Link className="navbar-item" to="/projects">
-              View All Projects
+            <Link className="navbar-item" to="/browse">
+              Browse Projects
```

Comment:
Ditto for this - let's just keep calling the link "Projects" - the user will still certainly go there to browse.